ansible/roles/touchforms/templates/localsettings.py.j2

```diff
@@ -3,7 +3,8 @@
 #
 import os
 
-URL_ROOT = "https://{{ SITE_HOST }}/a/{{ '{{' }}DOMAIN{{ '}}' }}"
+URL_HOST = "https://{{ SITE_HOST }}"
+URL_ROOT = "%s/a/{{ '{{' }}DOMAIN{{ '}}' }}" % URL_HOST
```

Comment:
lol i honestly have no clue why it is that. probably should figure that out at some point. i always thought it was some sort of magic